CHANGELOG.md

```diff
@@ -1,7 +1,18 @@
+# 5.0.0-beta.0 (2021-07-08)
+
+- Initial beta release.
+- Added support for `5.0.0-beta.*` versions of SKY UX component libraries. [#269](https://github.com/blackbaud/skyux-flyout/pull/269)
+- Fixed `SkyFlyoutService` to work within lazy-loaded feature modules. [#115](https://github.com/blackbaud/skyux-flyout/pull/115)
+
 # 5.0.0-alpha.0 (2021-05-21)
 
+- Added support for `@angular/core@^12`. [#114](https://github.com/blackbaud/skyux-flyout/pull/114)
 - Removed `BrowserAnimationsModule` from the `imports` section of `SkyFlyoutModule` to support lazy-loading. Consumers of `SkyFlyoutModule` must now import `BrowserAnimationsModule` into their application's root module. [#114](https://github.com/blackbaud/skyux-flyout/pull/114)
 
+# 4.2.0 (2021-06-10)
```

Comment:
Should this entry be placed above the `5.0.0-alpha.0` entry?

Reply:
Done.